src/db.js

```diff
@@ -1,18 +1,31 @@
 const flatfile = require('flat-file-db')
```

Comment:
should we wait to require this when we are going to use it? no need to require it if we are going to apply an adaptor